prusti-contracts/prusti-contracts-std/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "prusti-contracts-std"
+version = "0.1.0"
+authors = ["Vytautas Astrauskas <vastrauskas@gmail.com>"]
```

Comment:
Is everything Prusti-related authored by Vytautas? :) We might want to put something standard in all our crates, e.g. "Prusti contributors" + a mailing list.

Reply:
Haha I was wondering if someone would notice this. I agree, but we don't have such a mailing list yet right?